target/scripts/startup/setup-stack.sh

```diff
@@ -67,20 +67,22 @@ function _setup_default_vars
   PFLOGSUMM_RECIPIENT="${PFLOGSUMM_RECIPIENT:=${REPORT_RECIPIENT}}"
   LOGWATCH_RECIPIENT="${LOGWATCH_RECIPIENT:=${REPORT_RECIPIENT}}"
 
+  VARS[LOGWATCH_RECIPIENT]="${LOGWATCH_RECIPIENT}"
+  VARS[PFLOGSUMM_RECIPIENT]="${PFLOGSUMM_RECIPIENT}"
+  VARS[PFLOGSUMM_SENDER]="${PFLOGSUMM_SENDER}"
+  VARS[PFLOGSUMM_TRIGGER]="${PFLOGSUMM_TRIGGER}"
+  VARS[POSTMASTER_ADDRESS]="${POSTMASTER_ADDRESS}"
+  VARS[REPORT_RECIPIENT]="${REPORT_RECIPIENT}"
+  VARS[REPORT_SENDER]="${REPORT_SENDER}"
+
+  : >/root/.bashrc
+  : >/etc/dms-settings
```

Comment:
Might be worth adding some inline documentation comment about why vars are being stored slightly differently in two different files.

`git blame` history should work technically, so up to you I'm just thinking forward with future maintainers that come across this, especially if any future refactorings shift code around (_eg: when `start-mailserver.sh` gutted out into multiple files and heavy revisions early this year, it makes tracing the origin of some things via `git blame` a bit more of a hassle_ :sweat_smile: ).

Reply:
[`f014754` (#2264)](https://github.com/docker-mailserver/docker-mailserver/pull/2264/commits/f014754300d31a199be1966d856b06d3146d4dd8)